README.md

```diff
@@ -1,91 +1,129 @@
+| Project oriented workflow | [wtf-project-oriented-workflow-slides](https://github.com/rstats-wtf/wtf-project-oriented-workflow-slides) (quarto) | [wtf-explore-libraries](https://github.com/rstats-wtf/wtf-explore-libraries), [wtf-fix-paths](https://github.com/rstats-wtf/wtf-fix-paths), [wtf-packages-report](https://github.com/rstats-wtf/wtf-packages-report) |
+| Debugging                 | [wtf-debugging-slides](https://github.com/rstats-wtf/wtf-debugging-slides) (quarto)                                 | [wtf-debugging](https://github.com/rstats-wtf/wtf-debugging), associated pkg [wtfdbg](https://github.com/rstats-wtf/wtfdbg)                                                                                             |
+| Git/GitHub                | [drive assests](https://drive.google.com/drive/u/0/folders/12shxFDi5SPjwS6vc8UVIJRJZOQoeViV9) (keynote)              | In slides                                                                                                                                                                                                                   |
+| Personal R Administration | [wtf-personal-radmin-slides](https://github.com/rstats-wtf/wtf-personal-radmin-slides) (quarto)                     | In slides                                                                                                                                                                                                                   |
+| Iterating well with purrr | [wtf-purrr-slides](https://github.com/rstats-wtf/wtf-purrr-slides) (quarto)                                         | In slides                                                                                                                                                                                                                   |
+| Closing slides            | [drive assests](https://drive.google.com/drive/u/0/folders/12shxFDi5SPjwS6vc8UVIJRJZOQoeViV9) (google slides)         | None                                                                                                                                                                                                                        |
+# Notes
+
 This repo has two purposes:
 
-  1. A place to record policies about how to run an instance of the "What They Forgot ..." (WTF) workshop, such as what to store where.
-  1. Central storage for reusable assets that don't belong any where else, such as logos or diagrams.
-  
+1.  A place to record policies about how to run an instance of the "What
+    They Forgot ..." (WTF) workshop, such as what to store where.
+2.  Central storage for reusable assets that don't belong any where
+    else, such as logos or diagrams.
+
 Main types of content and where we store it:
 
-  * [rstats.wtf](https://rstats.wtf) is a bookdown site where we aspire to
```

Comment:
Kind of a bummer that the whitespace changes makes this big diff. I don't think anything changed in this section, yeah?

In hindsight, using the visual editor with this file on `main` first and committing the whitespace changes would be a better basis for the PR. But I also don't think it matters that much in this situation.

But if there **are** real changes here, I think it's worth going back and doing this, so we can see those updates.

Reply:
Nope, there aren't real changes to the content here. I thought it looked funny, too.